### PR TITLE
Examples on arrays should be directly on the property, not on the item

### PR DESCRIPTION
**Pull request by collinsauve**
Fixes an issue with adding examples and defaults to properties that are arrays.

# When I do...

```
expose :emailAddresses,
       documentation: {
         type: String,
         is_array: true,
         example: ['jane.doe@acme.co', 'janedoe@gmail.com'],
       }
```

# I get ...

This results in Swagger doc like this:

```
"emailAddresses": {
  "type": "array",
  "items": {
    "type": "string",
    "example": [
      "jane.doe@acme.co",
      "janedoe@gmail.com"
    ]
  }
},
```

Showing this in Redoc:

image

# But I expect ...

This is setting it as an *array of arrays*, which is not correct.  I expect to have the example directly on `emailAddresses`, not on `emailAddresses.items`.

This PR fixes that, and will now produce:

```
"emailAddresses": {
  "type": "array",
  "items": {
    "type": "string"
  },
  "example": [
    "jane.doe@acme.co",
    "janedoe@gmail.com"
  ],          
},
```

image

### CODE REVIEW

**Comment by mscrivo** [comment]
@dblock Who's the maintainer of this repo? @collinsauve is a colleague of mine, we're working on the same project. Would be nice to get this in. @collinsauve Can we add a test to verify the new behaviour?

**Comment by dblock** [comment]
@mscrivo for this PR, right? That's a different story, we can take care of that.

**Comment by dblock** [comment]
Thanks for #61 @mscrivo, @collinsauve want to rebase this?

**Comment by mscrivo** [comment]
> Thanks for #61 @mscrivo, @collinsauve want to rebase this?

done and added a test that fails with the previous code.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,17 @@
 
 #### Fixes
 
+* [#60](https://github.com/ruby-grape/grape-swagger-entity/pull/60): Examples on arrays should be directly on the property, not on the item - [@collinsauve](https://github.com/collinsauve).
 * [#61](https://github.com/ruby-grape/grape-swagger-entity/pull/61): Migrate from Travis to GHA for CI - [@mscrivo](https://github.com/mscrivo).
 * Your contribution here.
 
-### 0.5.1 (June 30, 2020)
```
Comment (dblock) [inline CHANGELOG.md:12]: Getting old!

**Review by dblock — APPROVED** [review]

**Pull request merged**